pom.xml

```diff
@@ -277,6 +251,15 @@
                             <rpt>SCRIPT_STYLE</rpt>
                         </mapping>
                     </configuration>
+                    <dependencies>
+                        <dependency>
+                            <!-- includes common copyright headers for sources -->
+                            <groupId>org.kaazing</groupId>
+                            <artifactId>community.license</artifactId>
+                            <type>jar</type>
+                            <version>${project.version}</version>
```

Comment:
Yes, I was wondering about the early-binding vs. late-binding on this one, but only had `develop-SNAPSHOT` version during (successful) local testing.

Is the release process smart enough to replace an explicit reference to `develop-SNAPSHOT` but not `${project.version}` ?